bin/src/main.rs

```diff
@@ -66,6 +66,14 @@ fn main() -> anyhow::Result<()> {
         }
     }
 
+    #[cfg(any(target_os = "macos", target_os = "linux"))]
+    {
+        info!(
+            "Open Files limits: {:?}",
```

Comment:
Nit: Capitalization and pluralization seems a bit inconsistent. This limit also sounds like an application limit so I think we might be able to clarify it's a system limit.

e.g.
```
Detected system max open files:
```